nixos/modules/services/databases/mysql.nix

```diff
@@ -21,6 +21,17 @@ let
   installOptions =
     "${mysqldOptions} ${lib.optionalString isMysqlAtLeast57 "--insecure"}";
 
+  # slight modification of lib.generators.mkKeyValueDefault: bit of a hack until lib.generators.toINI supports lists
+  mkKeyValueMysql = {
```

Comment:
@Infinisil what are the chances `toINI` is going to get list handling built right in?

Reply:
https://github.com/NixOS/nixpkgs/pull/82208 :)